packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -65,6 +69,10 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
         return this.container.clientId;
     }
 
+    public async attach() {
```

Comment:
We may want to let the attach function to be included in the constructor for `FluidContainer`. For example, `OdspClient` would require a specially formatted request including the siteUrl, driveId, folderName, etc. Each client could provide the service specific route required for attaching the container.

Reply:
It's not going to be just the url for every container. By including it in the constructor, each client can define how the  container should be attached

Reply:
good idea

Reply:
resolved